File: SCRIPTS/2.4/metodo_inicializador.py
class MetodoIniciador:
    def __init__(self, nome, idade):
        self.nome = nome
        self.idade = idade
    def __str__(self):
        return f'nome: {self.nome}, idade: {self.idade}'

teste = MetodoIniciador('LUCAS', '26')
print(teste)

File: SCRIPTS/2.4/metodo_inicializador.php
<?php
class MetodoIniciador {
    private $nome;
    private $idade;
    function __construct($nome, $idade) {
        $this->nome = $nome;
        $this->idade = $idade;
    }

    public function str() {
        print "nome: $this->nome, idade: $this->idade";
    }
}

$teste = new MetodoIniciador('lucas', '23');

$teste->str();
?>